rq/config.py

```diff
@@ -0,0 +1,66 @@
+from redis import Redis
+from .utils import resolve_class
+from .serializers import resolve_serializer
+from .defaults import (DEFAULT_JOB_CLASS, DEFAULT_QUEUE_CLASS,
+                       DEFAULT_WORKER_CLASS, DEFAULT_SERIALIZER_CLASS)
+
+
+def _as_is(val):
+    return val
+
+
+_CONFIGURATIONS = {
+    'job_class': resolve_class,
+    'queue_class': resolve_class,
+    'worker_class': resolve_class,
+    'serializer': resolve_serializer,
+    'connection': _as_is
+}
+
+
+class Config:
+    def __init__(self, template=None, **kwargs):
```

Comment:
I'd like to have a more explicitly define config object so we can have an API like this:

```python
config = get_config()
config.serializer_class
config.connection
```